third_party/BUILD

```diff
@@ -623,6 +623,22 @@ filegroup(
     ],
 )
 
+genrule(
+    name = "java_tools_dist",
+    srcs = [
+        "//src/java_tools/buildjar:srcs",
+        "//src/java_tools/junitrunner:srcs",
+        "//src/java_tools/singlejar:srcs",
+        "//third_party/jarjar:srcs",
+        "//third_party/java/jdk/langtools:license-and-srcs",
+        "//third_party/zlib:embedded_tools",
+        "//third_party/ijar:srcs_with_deps",
+    ],
+    outs = ["java_tools_dist.zip"],
+    cmd = "zip -Xr $@ $$(echo $(SRCS) | sort)",
```

Comment:
nit: +`-9`?

Reply:
I tried archiving using `-9` and it doesn't make a difference...